content/community/action.md

```diff
@@ -6,32 +6,113 @@ image: "/images/learn/innersourceinaction.png"
-        [Tencent] adopted InnerSource to promote engineer culture inside the company... 80% of  projects get InnerSourced.
+      [Tencent] adopted InnerSource to promote engineer culture inside the company... 80% of  projects get InnerSourced.
+      {{< /company >}}
+      {{< company name="The Hartford" image="/images/logos/thehartford.png" link="https://www.youtube.com/watch?v=JCHbtKu71SQ" author_name="Jo Ann Tan" author_title="Vice President and Head of Infrastructure" >}}
+      Our development teams have benefited from the collaborative features of GitHub. It allows them to take co-development to a whole new level by working as a community and maturing our InnerSourcing practices.
+      {{< /company >}}
+       {{< company name="Tray.io" image="/images/logos/tray.png" link="https://github.com/customer-stories/trayio" author_name="Alberto Giorgi" author_title="Director of Engineeering" >}}
+      Anyone can learn something from another team and quickly see what’s in the code..Having a fairly open structure allows people to get involved quite easily and quickly.
+      {{< /company >}}
+      {{< company name="Trustpilot" image="/images/logos/trustpilot.png" link="https://github.com/customer-stories/trustpilot" author_name="Jo Ann Tan" author_title="Vice President and Head of Infrastructure" >}}
+      We have an InnerSource policy, Everybody has access to create pull requests on all repositories. You can see all our code. We don’t lock down anything.
+      {{< /company >}}
+       {{< company name="Twilio" image="/images/logos/twilio.png" link="https://www.youtube.com/watch?v=JCHbtKu71SQ" author_name="Jo Ann Tan" author_title="Vice President and Head of Infrastructure" >}}
+      Our development teams have benefited from the collaborative features of GitHub. It allows them to take co-development to a whole new level by working as a community and maturing our InnerSourcing practices.
       {{< /company >}}
       {{< company name="Vmware" image="/images/logos/vmware.png" link="https://www.youtube.com/watch?v=Y6biezNfJOM" author_name="Minchene Tang" author_title="Vmware" >}}
-        A project isn’t just about the code, there’s a team behind it, including writers, designers and project managers. [InnerSource] gives you the opportunity to collaborate across multiple time zones and business units.
+      A project isn’t just about the code, there’s a team behind it, including writers, designers and project managers. [InnerSource] gives you the opportunity to collaborate across multiple time zones and business units.
+      {{< /company >}}
+      {{< company name="Walmart" image="/images/logos/walmart.png" link="https://resources.github.com/whitepapers/introduction-to-innersource/" author_name="Jeremy King" author_title="Executive Vice President & Chief Technology Officer for Global eCommerce" >}}
```

Comment:
IMHO, the title is too long and can break the layout. Can we shorten `Chief Technology Officer` -> `CTO`?